src/Apps/Order2/Routes/Details/Components/__tests__/Order2DetailsPage.jest.tsx

```diff
@@ -36,6 +36,6 @@ const orderData = {
   internalID: "a5aaa8b0-93ff-4f2a-8bb3-9589f378d229",
   code: "123",
   displayTexts: {
-    titleText: "Test Order Title",
+    title: "Test Order Title",
```

Comment:
Would it make sense to test the message as well?

Reply:
Yes! I'll update the test once I figure out how to properly link things in this text. That is the part that is still left on this card.